demo-apps/angular/angular-app/src/app/app.component.html

```diff
@@ -0,0 +1,27 @@
+
+<style>
+
+  main {
+    width: 100%;
+    min-height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 1rem;
+    box-sizing: inherit;
+    position: relative;
+  }
+
```

Comment:
We can remove this whole style section - since we are using gcds-container, none of these styles will be applied.

Reply:
Done!